src/test/java/io/appium/java_client/internal/AppiumUserAgentFilterTest.java

```diff
@@ -0,0 +1,20 @@
+package io.appium.java_client.internal;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.appium.java_client.AppiumUserAgentFilter;
+import org.junit.jupiter.api.Test;
+
+public class AppiumUserAgentFilterTest {
+    @Test
+    void validateUserAgent() {
+        assertTrue(AppiumUserAgentFilter.USER_AGENT.startsWith("appium/"));
+    }
+
+    @Test
+    void validUserAgentIfContainsAppiumName() {
+        assertFalse(AppiumUserAgentFilter.USER_AGENT.startsWith("selenium/4.5.0 (java mac)"));
+        assertTrue(AppiumUserAgentFilter.USER_AGENT.startsWith("appium/8.2.0 (selenium/4.5.0 (java mac))"));
```

Comment:
most probably each dependencies version bump or each release will require to update this test, does it make sense to use regular expressions here in order to validate the format?

Reply:
Sorry, this test was wrong, just wanted to test containsAppiumName method (updated in the latest code)